applications/StructuralMechanicsApplication/custom_python/structural_mechanics_python_application.cpp

```diff
@@ -150,6 +150,8 @@ PYBIND11_MODULE(KratosStructuralMechanicsApplication,m)
 
     // Mixed formulations variables
     KRATOS_REGISTER_IN_PYTHON_VARIABLE(m, REACTION_STRAIN )
+    KRATOS_REGISTER_IN_PYTHON_SYMMETRIC_3D_TENSOR_VARIABLE_WITH_COMPONENTS(m, NODAL_STRAIN_VECTOR )
```

Comment:
I'd remove the `NODAL_` tag from the variable name so we can reuse it for other stuff. Besides, after doing a grep in Kratos, I found a `STRAIN_VECTOR` variable in your FEM-DEM application, so maybe it's easier to move that variable to the core and use it also here.

Reply:
So as as far as I see, you created a new `STRAIN_VECTOR` variable in the `KratosCore` but you did not remove the `FemToDemApplication` one. Indeed, this (if I'm not wrong) should be a linking error.

I'd do a standalone PR just

Reply:
After a brief conversation with @AlejandroCornejo, the idea is to upgrade the FemToDem application in a different PR.